corehq/apps/zapier/tests.py

```diff
@@ -396,3 +397,122 @@ def test_case_fields(self):
         actual_fields = ZapierCustomFieldCaseResource().obj_get_list(bundle)
         for i in range(len(actual_fields)):
             self.assertEqual(expected_fields[i], actual_fields[i].get_content())
+
+
+class TestZapierCreateCaseAction(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestZapierCreateCaseAction, cls).setUpClass()
+        cls.test_url = "http://commcarehq.org/?domain=fruit&case_type=watermelon&user_id=test_user&user=test"
+        cls.data = {'case_name': 'test1', 'price': '11'}
+        cls.accessor = CaseAccessors('fruit')
+        cls.domain_object = Domain.get_or_create_with_name('fruit', is_active=True)
+        cls.domain = cls.domain_object.name
+        cls.user = WebUser.create(cls.domain, 'test', '******')
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.user.delete()
+        cls.domain_object.delete()
+        super(TestZapierCreateCaseAction, cls).tearDownClass()
+
+    def test_create_case(self):
+        factory = RequestFactory()
+        request = factory.post(self.test_url,
+                              data=json.dumps(self.data),
+                              content_type='application/json')
+
+        status = ZapierCreateCase().post(request)
+        self.assertEqual(status.status_code, 200)
+
+        case_id = self.accessor.get_case_ids_in_domain()
```

Comment:
would it ever be useful to return the case_id of the new case in the HTTP response? I could see this being useful if Zapier shows any information about successful responses in their dashboard.